components/authz-service/server/v2/policy.go

```diff
@@ -94,6 +94,10 @@ func (s *policyServer) CreatePolicy(
 	req *api.CreatePolicyReq) (*api.Policy, error) {
 
 	// API requests always create custom policies.
+	err := validateRequiredFieldsAndProjectsForPolicy(req.Id, req.Name, req.Projects)
+	if err != nil {
+		return nil, status.Errorf(codes.InvalidArgument, err.Error())
```

Comment:
[nit] There's nothing to format here, so we could use [`status.Error`](https://pkg.go.dev/google.golang.org/grpc/status?tab=doc#Error):
```suggestion
		return nil, status.Error(codes.InvalidArgument, err.Error())
```

(The same applies below.)